src/composables/multisigAccounts.ts

```diff
@@ -164,6 +164,7 @@ export function useMultisigAccounts({ store }: UseMultisigAccountsOptions) {
         }
         consensus.expiration_height = Number(consensus.expiration_height);
         consensus.confirmations_required = Number(consensus.confirmations_required);
+        consensus.totalConfirmations = Number(consensus.confirmed_by.length);
```

Comment:
We already have a totalConfirmations in the `multisigAccount` object.

Reply:
Removed it.